docs/sources/alerting/rules.md

```diff
@@ -88,6 +88,13 @@ So as you can see from the above scenario Grafana will not send out notification
 to fire if the rule already is in state `Alerting`. To improve support for queries that return multiple series
 we plan to track state **per series** in a future release.
 
+> Starting with Grafana v5.3 you can configure reminders to be sent for triggered alerts. This will send additional notifications
+> when an alert continues to fire. If other series cause the alert they'll be included in the reminder notification. Depending on
```

Comment:
"If other series cause the alert" is a bit unclear. Maybe "If other series (like **server2** in the example above) also cause the alert rule to fire they will be included..."